### ISSUE TITLE
[feat]: installation

### ISSUE THREAD

**Issue by xbozo** [sadmann7/file-uploader#111]
### Feature description

would love to see a installation command to this, such as shadcn actually does with components - you run it and it automatically install dependencies libs, create the tsx files and make the hook documentation on the website

### Additional Context

makes it easier to use

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues and PRs

**Comment by Keith-Web3** [comment]
It would be nice if this could be implemented. I would have loved to use this component in my current project but there's not enough information on how to get it running.

**Comment by nihirv** [comment]
agreed. Nice looking component, but seems easier to code this from scratch than finding out how to implement the project in the repo

**Comment by sadmann7** [comment]
[ui library](https://github.com/sadmann7/shadcn-table/discussions/472)

**Comment by Roman-Ka** [comment]
Would've integrated it into our app straight away if it could be installed like chadcn, the uploader looks so good! Would be keeping up with updated @sadmann7 , thanks a lot for your great work!
